### PR TITLE
Clean up and Refactor "chia show" command

### PR DESCRIPTION
**Pull request by jack60612**
Making code readable again

### CODE REVIEW

**Comment by jack60612** [comment]
also removing some things that should not be there

File: chia/cmds/show.py [hunk 1]
```diff
@@ -1,13 +1,17 @@
-from typing import Any, Optional, Union, Dict
+import time
+from typing import Any, Callable, Dict, List, Optional, Union
 
-from chia.types.blockchain_format.sized_bytes import bytes32
 import click
 
-from chia.util.network import is_trusted_inner
+from chia.consensus.block_record import BlockRecord
```
Comment (mariano54) [inline chia/cmds/show.py:6]: we are importing extra stuff here. Can we import it inside the functions instead for efficiency?

**Review by mariano54 — COMMENTED** [review]
Overall looks like a nice improvement!

File: chia/cmds/show.py (hunk 1 above)
Comment (jack60612) [reply chia/cmds/show.py:6]: Yep

File: chia/cmds/show.py [hunk 2]
```diff
@@ -57,236 +61,253 @@ async def print_connections(client, time, NodeType, trusted_peers: Dict):
...
+            ip, port = (
+                ":".join(add_connection.split(":")[:-1]),
+                add_connection.split(":")[-1],
+            )
+            print(f"Connecting to {ip}, {port}")
+            try:
+                await node_client.open_connection(ip, int(port))
+            except Exception:
+                print(f"Failed to connect to {ip}:{port}")
+    if remove_connection:
```
[the remark this answers is not in the document]
Comment (jack60612) [reply chia/cmds/show.py:314]: They are all factored out now

**Review by mariano54 — APPROVED** [review]

**Pull request merged**